### DISCUSSION TITLE
Smooth Scrolling by default

### DISCUSSION THREAD

**Discussion by SaltyAimbOtter** [squidfunk/mkdocs-material#2128] [Ideas]
Hey 👋 

I just realized smooth scrooling exists.
So I wanted to open a feature request about that presumingly super-complex insider feature 🤯

Then I googled it and realized it's apparently super easy to add. Infact, I did even mange to add it myself using this snippet from w3schools & the extra CSS feature: 
```CSS
html {
  scroll-behavior: smooth;
}
```
https://www.w3schools.com/howto/howto_css_smooth_scroll.asp#section2

Maybe you should make this a config option / a default setting? I don't think a lot of people are aware of it. 
It really helps with the naviagtion on a single side when using a lot of links.
Tweaking this to not smooth-scroll between pages would also be nice.

**Comment by squidfunk** [comment]
It's indeed very simple to add, which is one reason why it's not enabled by default. For other reasons see https://github.com/squidfunk/mkdocs-material/issues/1011#issuecomment-467798043 😊
